src/js/components/AppPageComponent.jsx

```diff
@@ -182,6 +182,26 @@ var AppPageComponent = React.createClass({
     }
   },
 
+  getUnhealthyTaskMessage: function (healthCheckResults = []) {
+    return healthCheckResults.map((hc, index) => {
+      if (hc && !hc.alive) {
+        var failedCheck = this.state.app.healthChecks[index];
+
+        var protocol = failedCheck.protocol
+          ? `${failedCheck.protocol} `
```

Comment:
I suppose because we want a space between the protocol and the host/path part of the string: 

> "Warning: Health check 'HTTP 127.0.0.1/' failed".